cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,15 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.get(':nth-child(2) > .nav-link').click();
```

Comment:
Using ':nth-child' selectors is generally brittle because the structure of the HTML can easily change. Use data attributes like 'data-cy' for more stable selectors.